### ISSUE TITLE
Add New C14 Track Type "Drum Track"

### ISSUE THREAD

**Issue by mbfranck** [bjoluc/cubase-mcu-midiremote#56]
Currently, Cubase 14 drum tracks neither show up nor can be set to true or false in script preferences.

**Comment by bjoluc** [comment]
Right, I think it's currently excluded because the script whitelists enabled track types instead of blacklisting disabled ones. I'll fix this in the next release!

**Comment by bjoluc** [comment]
Done and released in [v1.11.0](https://github.com/bjoluc/cubase-mcu-midiremote/releases) :+1:
